File: src/server/Modules/Organization/Modules.Organization.Core/Enums/EarlyArrivalPolicy.cs
namespace FluentPOS.Modules.Organization.Core.Enums
{
    public enum EarlyArrivalPolicy
    {
        /// <summary>
        /// ACTUAL TIME: THE SYSTEM CALCULATES THE WORKING HOURS OF AN EMPLOYEE AS SOON AS HE/SHE MARKS THEIR ATTENDANCE.
        /// </summary>
        ActualTime = 1,

        /// <summary>
        /// SHIFT TIME: THE SYSTEM CALCULATES THE WORKING HOURS OF AN EMPLOYEE AFTER HIS/HER SHIFT STARTS.
        /// </summary>
        ShiftTime = 2,
    }
}

File: src/client/src/app/core/enums/EarlyArrivalPolicy.ts
export enum EarlyArrivalPolicy {
    ActualTime = 1,
    ShiftTime = 2
}
export const EarlyArrivalPolicyMapping: Record<EarlyArrivalPolicy, string> = {
    [EarlyArrivalPolicy.ActualTime]: "Actual Time",
    [EarlyArrivalPolicy.ShiftTime]: "Shift Time",
};